engine/table/src/main/java/io/deephaven/engine/table/impl/sources/UnionSourceManager.java

```diff
@@ -209,6 +212,22 @@ private final class ConstituentListenerRecorder extends LinkedListenerRecorder {
             setMergedListener(mergedListener);
         }
 
+        @Override
+        protected void onFailureInternal(@NotNull final Throwable originalException,
+                @Nullable final Entry sourceEntry) {
+            // We will just record the error here for now. If the table was removed, then we don't actually care about
+            // it
+            // but if the error is real, and the table is not removed, it will be propagated in processExisting()
```

Comment:
The suggestion is just cleanup, but I think the comment and code could use a re-write.

```suggestion
        protected void onFailureInternal(
                @NotNull final Throwable originalException,
                @Nullable final Entry sourceEntry) {
            // We will just record the error here for now. If the table was removed, then we don't actually care about
            // it but if the error is real, and the table is not removed, it will be propagated in processExisting()
```